package.json

```diff
@@ -39,7 +40,7 @@
   "dependencies": {
     "classnames": "~2.2.5",
     "prop-types": "~15.6.0",
-    "react-cosmos-utils": "~2.0.0",
+    "react-cosmos-shared": "^3.6.1",
```

Comment:
Commenting here because I can't on the actual unchanged line: we should change the peer dep to match the cosmos@3 package.